change-notes/1.25/analysis-csharp.md

```diff
@@ -24,5 +24,28 @@ The following changes in version 1.25 affect C# analysis in all applications.
   have type parameters. This means that non-generic nested types inside construced types,
   such as `A<int>.B`, no longer are considered unbound generics. (Such nested types do,
   however, still have relevant `.getSourceDeclaration()`s, for example `A<>.B`.)
+* The data-flow library has been improved, which affects and improves most security queries. Flow
+  through methods now takes nested field reads/writes into account. For example, the library is
+  able to track flow from `"taint"` to `Sink()` via the method `GetF2F1()` in
+  ```csharp
+  class C1
```

Comment:
I think the example might read better if `C1` and `F1` could be replaced by real-world examples like `Person` and `Surname`. Just an idea.